From 81466bb79c403a6afc60f36335997cc9675bd919 Mon Sep 17 00:00:00 2001
From: Traian Schiau <traianx.schiau@intel.com>
Date: Thu, 7 Jan 2016 10:23:19 +0200
Subject: [PATCH 0732/1214] intel_th-dvc: Fix deactivation issue.

mdd->th_dev should not be reset upon dvc-source device
deactivation.

Signed-off-by: Traian Schiau <traianx.schiau@intel.com>
---
 drivers/hwtracing/intel_th/msu-dvc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/hwtracing/intel_th/msu-dvc.c b/drivers/hwtracing/intel_th/msu-dvc.c
index db8d877..b8c7001 100644
--- a/drivers/hwtracing/intel_th/msu-dvc.c
+++ b/drivers/hwtracing/intel_th/msu-dvc.c
@@ -925,7 +925,6 @@ static void mdd_deactivate(struct dvct_source_device *client)
 	struct msu_dvc_dev *mdd = dtc_to_mdd(client);
 
 	MDD_F_DEBUG();
-	mdd->th_dev = NULL;
 	mdd->dtc_status = NULL;
 }
 
-- 
2.7.4

